subprojects/model-core/src/test/groovy/org/gradle/internal/reflect/validation/ValidationMessageCheckerTest.groovy

```diff
@@ -54,6 +54,22 @@ Possible solutions:
   1. Assign a value to 'someProperty'.
   2. Mark property 'someProperty' as optional.
 """
+        when:
+        render(missingNonConfigurableValueMessage() {
+            type('SomeType')
+            property('someProperty')
+        })
+
+        then:
+        outputEquals """
+Type 'SomeType' property 'someProperty' doesn't have a configured value.
```

Comment:
🤔 Should type and property name be reversed, e.g.:
`Property 'someProperty' of type 'SomeType'...
?

Reply:
More  I read, more confusing it is that type is here first